docs/guides/user-guide/general-faqs.md

```diff
@@ -0,0 +1,189 @@
+
+### What is the Personal+ Plan?
+
+The Personal+ Plan (formerly known as the Astronaut Plan) allows users to create unlimited private projects in both Classic and Codeflow editors, as well as unlimited file uploads. Personal+ users can also use Codeflow technology for private repositories.
+
+This plan is ideal for individual contributors, students, or developers working on singular projects, or engineering candidates. Users focusing on bug reproductions, fast prototyping, and sharing projects will find this plan sufficient.
+
+## Teams Plan
+
+### What is the Teams plan?
+
+The Teams plan (currently in Beta) is a paid membership offering that allows users to share features amongst members of their GitHub organization. Features and highlights of this plan include:
+
+- Codeflow access for all users in the GitHub organization (in-browser VS Code)
+- Instant integration with your GitHub organization’s repos for seamless in-browser editing
+    - Utilizing GitHub organization makes it super easy for all users in the organization to use the Codeflow editor and Codeflow bot to create and review pull requests.
+- A unified security model for sharing StackBlitz projects and GitHub repos
+- User management and billing linked to your GitHub organization
+    - Easy to use interface within the StackBlitz dashboard (easy to toggle users for access/no access, all-in-one monthly payment).
+
+If you're interested in Teams Beta, come onboard with us for a private technology preview of this plan! You can sign up [here](https://stackblitz.com/beta-teams-signup).
+
+## Enterprise Server
+
+### What is the Enterprise Server offering?
+
+Enterprise Server is a membership offering that's best fit for organizations who want to use StackBlitz behind their firewall. Benefits include:
+
+- Private NPM registry integration
+- Custom SSO integration
+- On-prem installation support
+- Dedicated Customer Success Engineer
+
+Check out the [Enterprise page](https://stackblitz.com/enterprise) for details and set up a call with our Enterprise team!
+
+### I want to purchase the enterprise edition for WebContainer. Where can I do that?
+
+Bringing Webcontainer to Enterprise Server is in the works, but not currently available. You can reach out to enterprise@stackblitz.com for details.
+
+We do offer [WebContainer API](webcontainers.io) licenses for organizations wanting to use WebContainer technology for production usage in a for-profit setting. Check out [webcontainers.io](webcontainers.io) for more information!
+
+# Technology FAQs
+
+## Classic Editor (Engineblock projects)
+
+### What is the Classic Editor for Engineblock projects?
+
+The Classic Editor utilizes Engineblock and WeBcontainer technologies to run. As the original StackBlitz interface, the Classic Editor has a file tree, an npm package manager, and a light dev server and preview pane that allows you to see your app as you code.
+
+Engineblock projects are distinguishable as they do not have a terminal.
```

Comment:
distinguishable FROM webcontainers-based projects. for more info re: webcontainers, <ADD LINK that goes down to webcontainer FAQ block>